### PR TITLE
feat: add execute proposal to proposal page

### PR DESCRIPTION
**Pull request by ryanchristo**
Closes: #130

### How to test

You may need to test locally and update the code to allow a shorter voting period unless you have a group with an accepted proposal ready to execute.

- Create group and policy with 30 second voting period
- Create proposal (any proposal, can be text proposal)
- Vote on proposal and wait for voting period to end
- View proposal page and notice execute footer button
- Execute the proposal using the execute footer button

### CODE REVIEW

File: src/pages/proposal-page.tsx [hunk 1]
```diff
@@ -42,6 +55,13 @@ export default function ProposalPage() {
     isLoading: isLoadingVotes,
     refetch: refetchVotes,
   } = useProposalVotes(proposalId)
+  const [submitting, setSubmitting] = useState(false)
+
+  useFormFooter({
+    onNext: undefined,
+    onPrev: undefined,
+    btnText: 'Execute',
+  })
```
Comment (ryanchristo) [inline src/pages/proposal-page.tsx:64]: For some reason this is not rendering correctly and the button text says `Submit`. When developing locally and making changes the button switches between `Submit` and `Execute`. Any ideas what's going on here?

**Comment by ryanchristo** [comment]
This is ready for review but still investigating what is causing the button text to not render correct on initial load.

File: src/pages/proposal-page.tsx (hunk 1 above)
Comment (wgwz) [reply src/pages/proposal-page.tsx:64]: is there like a way check that whatever `useFormFooter` is doing has completed before rendering anything? just a rough idea

**Review by wgwz — APPROVED** [review]
utACK. pre-approving because despite the bug this code LGTM and i trust your testing :-)

**Comment by ryanchristo** [comment]
I did a little more investing and opened a followup to address the issue: #137

**Pull request merged**